Cargo.toml

```diff
@@ -13,12 +13,17 @@ web-app = ["tokio", "axum", "axum-server", "hyper", "hyper-tls"]
 self-signed-certs = ["hyper-tls"]
 
 [dependencies]
+aes = "0.8"
+byteorder = "1"
 # rust-elgamal (via curve25519-dalek-ng) only works with digest 0.9, not 0.10
 digest = "0.9"
 hex = {version = "0.4", optional = true}
 # rust-elgamal (via curve25519-dalek-ng) only works with digest 0.9, so pin this
 hkdf = "0.11"
 log = "0.4"
+# This is stupid, but we have packages that want this old interface
+# those have to use the same RNG as packages that want the new interface.
+old_rand_core = { package = "rand_core", version = "0.5", default-features = false }
```

Comment:
should we use https://crates.io/crates/x25519-dalek/2.0.0-pre.1 that uses [0.6](https://github.com/dalek-cryptography/x25519-dalek/pull/64)?

Reply:
That might be easier than what I did, yeah.